docs/user_guides/projects/airflow/airflow.md

```diff
@@ -0,0 +1,99 @@
+---
+description: Documentation on how to orchestrate Hopsworks jobs using Apache Airflow
+--- 
+
+# Orchestrate Jobs using Apache Airflow
+
+## Introduction 
+
+Hopsworks jobs can be orchestrated using [Apache Airflow](https://airflow.apache.org/). You can define a Airflow DAG (Direct Acyclic Graph) containing the dependencies between Hopsworks jobs. 
+You can then schedule the DAG to be executed at a specific schedule using a [cron](https://en.wikipedia.org/wiki/Cron) expression.
+
+Airflow DAGs are defined as Python files. Within the Python file, different operators can be used to trigger different actions. Hopsworks provides an operator to execute jobs on Hopsworks and a sensor to wait for a specific job to finish.
+
+### Use Apache Airflow in Hopsworks 
+
+Hopsworks deployments include a deployment of Apache Airflow. You can access it from the Hopsworks UI by clicking on the _Airflow_ button on the left menu.
+
+Airfow is configured to enforce Role Based Access Control (RBAC) to the Airflow DAGs. Admin users on Hopsworks have access to all the DAGs in the deployment. Regular users can access all the DAGs of the projects they are member of.
+
+!!! note "Access Control"
+    Airflow does not have any knowledge of the Hopsworks project you are currently working on. As such, when opening the Airflow UI, you will see all the DAGs of the projects you are member of.
+
+#### Hopsworks DAG Builder 
+
+<figure>
+  <img src="../../../../assets/images/guides/airflow/airflow_dag_builder.png" alt="Airflow DAG Builder"/>
+  <figcaption>Airflow DAG Builder</figcaption>
+</figure>
+
+You can create a new Airflow DAG to orchestrate jobs using the Hopsworks DAG builder tool. Click on _New Workflow_ to create a new Airflow DAG. You should provide a name for the DAG as well as a schedule interval. You can define the schedule using the dropdown menus or by providing a cron expression.
+
+You can add to the DAG Hopsworks operators and sensors:
+
+- **Operator**: The operator is used to trigger a job execution. When configuring the operator you select the job you want to execute and you can optionally provide execution arguments. You can decide whether or not the operator should wait for the execution to be done. If you select the _wait_ option, the operator will block and Airflow will not execute any parallel task. In this case the Airflow task fails if the job fails. If you want to execute tasks in parallel, you should not select the _wait_ option but instead use the sensor. When configuring the operator, you can can also provide which other Airflow tasks it depends on. If you add a dependency, the task will be executed only after the upstream tasks have been executed successfully. 
```

Comment:
execution to be done -> execution to complete
In this case, 
specify which other Airflow tasks